modules/nrf_wifi/bus/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: BSD-3-Clause
```

Comment:
Hi, this has not been fixed and is still BSD-3-Clause upstream, @krish2718 would you be able to send a PR to fix it?

https://github.com/zephyrproject-rtos/zephyr/blob/c7a19da8ca1abefa8eac41e704e7c678af0282a9/modules/nrf_wifi/bus/CMakeLists.txt#L4

cc @kartben

Reply:
FYI https://github.com/zephyrproject-rtos/zephyr/pull/86748